README.md

```diff
@@ -1,7 +1,6 @@
 ![Ingress Controller - Status](https://img.shields.io/badge/project--status-beta-orange.svg)
-[![Build Status](https://travis-ci.org/Azure/application-gateway-kubernetes-ingress.svg?branch=master)](https://travis-ci.org/Azure/application-gateway-kubernetes-ingress)
+[![Build Status](https://dev.azure.com/azure/application-gateway-kubernetes-ingress/_apis/build/status/Old%20build%20pipeline?branchName=master)](https://dev.azure.com/azure/application-gateway-kubernetes-ingress/_build/latest?definitionId=26&branchName=master)
```

Comment:
[![Build Status](https://dev.azure.com/azure/application-gateway-kubernetes-ingress/_apis/build/status/Merge%20Builds?branchName=master)](https://dev.azure.com/azure/application-gateway-kubernetes-ingress/_build/latest?definitionId=30&branchName=master)

Let use this one!